public/javascript/index.js

```diff
@@ -2,22 +2,96 @@ const charactersAPI = new APIHandler('http://localhost:8000');
 
 window.addEventListener('load', () => {
   document.getElementById('fetch-all').addEventListener('click', function (event) {
-
+    charactersAPI
+      .getFullList()
+      .then(response => {
+        let text = ''
+        response.data.reverse().forEach(elm => text += 
+          `<div class="character-info">
+          <div class="name">${elm.name}</div>
+          <div class="occupation">${elm.occupation}</div>
+          <div class="cartoon">${elm.debt}</div>
+          <div class="weapon">${elm.weapon}</div>
+          <div class="weapon">${elm.id}</div>
+          </div>`
+          )
+        document.querySelector('.characters-container').innerHTML = text
+      })
   });
 
   document.getElementById('fetch-one').addEventListener('click', function (event) {
+     
+    const id = document.querySelector('#one').value
+
+    charactersAPI
+    .getOneRegister(id)
+    .then(response =>{
+
+      const{id, name, weapon, occupation, cartoon} = response.data
 
+      let text =
+         `<div class="character-info">
+          <div class="name">${name}</div>
+          <div class="occupation">${occupation}</div>
+          <div class="cartoon">${cartoon}</div>
+          <div class="id">${id}</div>
+          <div class="weapon">${weapon}</div>
+          </div>`
+          document.querySelector('.characters-container').innerHTML = text
+    })
   });
 
   document.getElementById('delete-one').addEventListener('click', function (event) {
+    
+    const id = parseInt(document.querySelector('#two').value)
 
+    charactersAPI
+    .deleteOneRegister(id)
+    .then()
+    console.log(charactersAPI.deleteOneRegister())
```

Comment:
No quiero ver nunca una promesa sin su then y su catch gestionados!!!

Estás perdiendo información y metiendote en una ratonera. Siempre deberías hacerle al menos un console.log a lo que salga de ahí.

```suggestion
    charactersAPI
    .deleteOneRegister(id)
    .then(response => console.log(response))
    .catch(err => console.log(err))
```